src/code/V2ServerAPICalls.cs

```diff
@@ -149,25 +149,30 @@ public override FindResults FindTags(string[] tags, bool includePrerelease, Reso
                 {
                     return new FindResults(stringResponse: responses.ToArray(), hashtableResponse: emptyHashResponses, responseType: v2FindResponseType);
                 }
-                responses.Add(initialScriptResponse);
+
                 int initalScriptCount = GetCountFromResponse(initialScriptResponse, out errRecord);
```

Comment:
```suggestion
                int initialScriptCount = GetCountFromResponse(initialScriptResponse, out errRecord);
```

typo in variable name